fitbenchmarking/fitbenchmarking
# Adds top level index page

#### Description of Work

Fixes #131 

Creates a top level index page from a template

#### Testing Instructions

1. Check index page is clear and understandable.
2. Check the buttons work

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
